actionpack/test/dispatch/mime_type_test.rb

```diff
@@ -30,67 +30,67 @@ class MimeTypeTest < ActiveSupport::TestCase
     accept = "text/*, text/html, application/json, multipart/form-data"
     expect = [Mime[:html], Mime[:text], Mime[:js], Mime[:css], Mime[:ics], Mime[:csv], Mime[:vcf], Mime[:xml], Mime[:yaml], Mime[:json], Mime[:multipart_form]]
     parsed = Mime::Type.parse(accept)
-    assert_equal expect, parsed
+    assert_mime_types_equal expect, parsed
   end
 
   test "parse text with trailing star in the end" do
     accept = "text/html, application/json, multipart/form-data, text/*"
     expect = [Mime[:html], Mime[:json], Mime[:multipart_form], Mime[:text], Mime[:js], Mime[:css], Mime[:ics], Mime[:csv], Mime[:vcf], Mime[:xml], Mime[:yaml]]
     parsed = Mime::Type.parse(accept)
-    assert_equal expect, parsed
+    assert_mime_types_equal expect, parsed
```

Comment:
This test needs to check the order since the accept header is in a specific order